pkg/sql/codegen/pai/pai.go

```diff
@@ -40,3 +40,18 @@ func getPAIPredictCmd(ir *ir.PredictStmt, session *pb.Session) (string, error) {
 	return fmt.Sprintf(`pai -name prediction -DmodelName="%s" -DinputTableName="%s" -DoutputTableName="%s" -DfeatureColNames="%s" -DappendColNames="%s"`,
 		ir.Using, ir.TmpPredictTable, ir.ResultTable, strings.Join(flds, ","), strings.Join(flds, ",")), nil
 }
+
+// CleanupPAIModel can drop saved PAI model
+func CleanupPAIModel(ir *ir.TrainStmt, session *pb.Session) error {
+	db, err := database.OpenAndConnectDB(session.DbConnStr)
+	if err != nil {
+		return err
+	}
+	if ir.Estimator == "kmeans" || ir.Estimator == "randomforests" {
```

Comment:
It looks like the `OpenAndConnectDB` could be reduced if the `ir.Estimator` is `"some_thing_else"`.